test/audits/internal-links.test.js

```diff
@@ -26,11 +26,13 @@ const AUDIT_RESULT_DATA = [
     url_to: 'https://www.example.com/article/dogs/breeds/choosing-an-irish-setter',
     url_from: 'https://www.example.com/article/dogs/just-for-fun/dogs-good-for-men-13-manly-masculine-dog-breeds',
     traffic_domain: 100,
+    priority: 0,
```

Comment:
in the implementation, `priority` is set to either `high`, `medium`, or `low`; but why it does end up as an integer here?

Reply:
This is a mistake - leftover code from initial implementation of priorities. I will take care of this while implementing test cases which is not done yet. thx